adafruit_ads7830/ads7830.py

```diff
@@ -0,0 +1,126 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 Liz Clark for Adafruit Industries
+#
+# SPDX-License-Identifier: MIT
+"""
+:py:class:`~adafruit_ads7830.ads7830.ADS7830`
+================================================================================
+
+CircuitPython driver for the ADS7830 analog to digital converter
+
+
+* Author(s): Liz Clark
+
+Implementation Notes
+--------------------
+
+**Hardware:**
+
+* `Adafruit ADS7830 8-Channel 8-Bit ADC with I2C <https://www.adafruit.com/product/5836>`_
+
+**Software and Dependencies:**
+
+* Adafruit CircuitPython firmware for the supported boards:
+  https://circuitpython.org/downloads
+
+* Adafruit's Bus Device library: https://github.com/adafruit/Adafruit_CircuitPython_BusDevice
+"""
+
+from adafruit_bus_device.i2c_device import I2CDevice
+from micropython import const
+
+try:
+    import typing  # pylint: disable=unused-import
+    from busio import I2C
+except ImportError:
+    pass
+
+__version__ = "0.0.0+auto.0"
+__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_ADS7830.git"
+
+_I2C_ADDR = const(0x48)
+
+
+class ADS7830:
+    """Adafruit ADS7830 ADC driver"""
+
+    POWER_DOWN_MODES = [
+        0x00,
+        0x01,
+        0x02,
+        0x03,
+    ]
+    """Power down modes
+
+    :param int 0: power down between conversions
+    :param int 1: internal reference off, ADC on
+    :param int 2: internal reference on, ADC off
+    :param int 3: internal reference on, ADC on (default)"""
```

Comment:
I'd probably move this into the `__init__` comment where people provide it. It might be simpler to split into two bools that you then map to a number internally.

Reply:
that makes sense, thank you!